tests/Byteflag.spec.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
```

Comment:
It turns on JavaScript's [strict mode](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode) -- it changes the behaviour slightly to prevent oopsies like declaring global variables, and throws errors instead of failing silently with certain actions. It's good practice to always use it and I plan on adding it to the existing code at some point in the near future.